src/vm/exceptmacros.h

```diff
@@ -334,14 +334,14 @@ VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHar
             UNREACHABLE();                                                                          \
         }
 
-#else
+#else // defined(FEATURE_PAL) && defined(WIN64EXCEPTIONS)
```

Comment:
A nit, the comment style we use is just FEATURE_PAL && WIN64EXCEPTIONS. The same at the #endif below.